packages/shared/components/atoms/boxes/AddressBox.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="typescript">
+    import Box from './Box.svelte'
+    import { Text } from 'shared/components'
+
+    export let address: string = ''
```

Comment:
Ahh I am so used to typing everything. I think there is some cases that need it because it can be various types, so I will go through the code and check 👍

Reply:
Fixed in this file